extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -883,9 +883,10 @@ export class AmpAnalytics extends AMP.BaseElement {
   /**
    * @param {string} request The full request string to send.
    * @param {!JsonObject} trigger
+   * @param {string} body The full request body to send as a stringified json.
    * @private
    */
-  sendRequest_(request, trigger) {
+  sendRequest_(request, trigger, body) {
```

Comment:
I don't see it get called, maybe I missed something.

Reply:
here:
https://github.com/ampproject/amphtml/pull/15729/files/73830011fd4fa1172b93cf7f09f3c7f7f69d6058#diff-b951eee683ce1cc97c5e22f9abd32f11R272